pkg/apis/kops/cluster.go

```diff
@@ -147,6 +148,9 @@ type ClusterSpec struct {
 
 	// IAM field adds control over the IAM security policies applied to resources
 	IAM *IAMSpec `json:"iam,omitempty"`
+	// Enable External Cloud Controller denotes if the external Cloud Controller Manager
+	// introduced in kuberentes 1.7 is enabled
+	EnableExternalCloudController bool
```

Comment:
If this is k-c-m specific, does it make sense to put it in the k-c-m struct?  Also, you need the json settings, and you need to generate api machinery.  Please see https://github.com/kubernetes/kops/blob/master/docs/development/api_updates.md for more details.  You missed a few steps :)

Reply:
Looking at the PR more, this appears to be more than just k-c-m.  Should we just have a pointer to CloudControllerManagerConfig?